packages/core/src/components/Status/Status.tsx

```diff
@@ -44,7 +44,12 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
     backgroundColor: theme.palette.status.pending,
   },
   failed: {
-    backgroundColor: 'rgba(245, 155, 35, 0.5)',
+    // Same as Error
```

Comment:
Any reason why not to use a single class for both? Since the different is going to just be the `aria-label`

Reply:
An Error and a Failure is logically two different things that can happen, so I see a need for keeping them both. 

But maybe that’s not what you meant?

Reply:
Not really, of course keeping both is needed, but if they're going to look the same anyway, why bother have multiple duplicated classes?

Reply:
Not I see what you mean :D Fixed! Thanks